src/Error/JsonApiExceptionRenderer.php

```diff
@@ -71,7 +71,7 @@ protected function _outputMessage($template)
         $stream->write($json);
 
         // set up the response
-        $this->controller->response
+        $this->controller->response = $this->controller->response
```

Comment:
`$this->controller->setResponse()`  right?

Reply:
both are true